### ISSUE TITLE
OLCF Titan ORTE not recognized correctly

### ISSUE THREAD

**Issue by jrossyra** [radical-cybertools/radical.pilot#1755]
This error came up for me today both in newly installed environments as well as an existing stable environment I haven't made changes to for a few months:

`  File "/lustre/atlas/scratch/jrossyra/bip149/radical.pilot.sandbox/rp.session.titan-ext1.jrossyra.017835.0017/pilot.0000/rp_install/lib/python2.7/site-packages/radical/pilot/agent/lm/orte.py", line 65, in lrms_config_hook
    assert(orte_info.get('version'))`

Here are the logs and stack output from a new installation:
```
  python               : 2.7.9
  pythonpath           : /sw/titan/.swci/0-login/opt/spack/20180315/linux-suse_linux11-x86_64/gcc-4.3.4/python-2.7.9-v6ctjewwdx6k2qs7ublexz7gnx457jo5/lib/python2.7/site-packages:/sw/xk6/xalt/0.7.5/site:/sw/xk6/xalt/0.7.5/libexec
  virtualenv           : /ccs/proj/bip149/admd/admdenv

  radical.pilot        : 0.50.16
  radical.utils        : 0.50.2
  saga                 : 0.50.1
```
[rp.session-logs.tar.gz](https://github.com/radical-cybertools/radical.pilot/files/2535590/rp.session-logs.tar.gz)

**Comment by jrossyra** [comment]
@andre-merzky just pinging on this issue, let me know if there's anything I can do to help move it along.

**Comment by jrossyra** [comment]
Fixed by #1768
